statsmodels/iolib/summary.py

```diff
@@ -494,7 +500,7 @@ def summary_params(results, yname=None, xname=None, alpha=.05, use_t=True,
                        [forg(std_err[i]) for i in exog_idx],
                        [forg(tvalues[i]) for i in exog_idx],
                        ["%#6.3f" % (pvalues[i]) for i in exog_idx],
-                       confint
+                       ci_lows, ci_highs 
```

Comment:
can this and the above be replaced by something like the case for tvalues
`["%#6.3f" % (confint[i,0]) for i in exog_idx],` ?

my guess, but I don't remember, is that all the min, max above was only necessary because there were two values in a column